mlir/test/Dialect/Vector/vector-contiguous-extract-strided-slice-to-extract.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: mlir-opt -split-input-file -test-vector-contiguous-extract-strided-slice-to-extract %s | FileCheck %s
```

Comment:
```suggestion
// RUN: mlir-opt --test-vector-contiguous-extract-strided-slice-to-extract %s | FileCheck %s
```

Reply:
Why not adding the `-----` separator instead?

Reply:
I don't understand. We would we add something that's not needed?

Reply:
It was so originally and switched to omitting splits following a review comment by @kuhar .

Reply:
My understanding is that each split can be tested in parallel by a different thread. That's the rationale that we've been using afaik.

Reply:
I don't think this is the case because the loop around it is serial?
https://github.com/llvm/llvm-project/blob/baa51ffd9c417c9ec272c601673c484bc4335319/mlir/lib/Support/ToolUtilities.cpp#L51-L90
If this is expected to be faster, it would be nice to have it documented in the developer docs. 

I thought it's mostly for unrecoverable errors (with verify-diagnostic) / repeated names that otherwise you wouldn't be able to execute within a single module / context.